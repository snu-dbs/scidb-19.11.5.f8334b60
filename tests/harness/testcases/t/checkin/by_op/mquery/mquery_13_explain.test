--setup
--start-query-logging
create array f <v:int64>[i=1:*]

--test

# Ignoring query output, only expect that this doesn't crash.
--start-igdata
_explain_logical('mquery(insert(build(<v:int64>[i=1:9],1*i),f))','afl')
_explain_physical('mquery(insert(build(<v:int64>[i=1:9],1*i),f))','afl')
--stop-igdata

--cleanup
remove(f)
--stop-query-logging
