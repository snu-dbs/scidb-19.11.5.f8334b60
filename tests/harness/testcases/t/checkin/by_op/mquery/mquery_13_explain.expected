SCIDB QUERY : <create array f <v:int64>[i=1:*]>
Query was executed successfully

SCIDB QUERY : <_explain_logical('mquery(insert(build(<v:int64>[i=1:9],1*i),f))','afl')>
[Query was executed successfully, ignoring data output by this query.]

SCIDB QUERY : <_explain_physical('mquery(insert(build(<v:int64>[i=1:9],1*i),f))','afl')>
[Query was executed successfully, ignoring data output by this query.]

SCIDB QUERY : <remove(f)>
Query was executed successfully

